pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1796,14 +1797,27 @@ func (dm *DockerManager) runContainerInPod(pod *v1.Pod, container *v1.Container,
 		return kubecontainer.ContainerID{}, fmt.Errorf("InspectContainer: %v", err)
 	}
 
-	// Create a symbolic link to the Docker container log file using a name which captures the
-	// full pod name, the container name and the Docker container ID. Cluster level logging will
-	// capture these symbolic filenames which can be used for search terms in Elasticsearch or for
-	// labels for Cloud Logging.
-	containerLogFile := containerInfo.LogPath
-	symlinkFile := LogSymlink(dm.containerLogsDir, kubecontainer.GetPodFullName(pod), container.Name, id.ID)
-	if err = dm.os.Symlink(containerLogFile, symlinkFile); err != nil {
-		glog.Errorf("Failed to create symbolic link to the log file of pod %q container %q: %v", format.Pod(pod), container.Name, err)
+	dockerLoggingDriver := dockerDefaultLoggingDriver
```

Comment:
Can we change the logic a bit?

```
if containerInfo.LogPath != ""; symlink;
else if loggingDriver == json-file; report an error;
else log some information.
```

So that if there are other logging drivers which populates `LogPath`, they could still work.